tests/snapshots/execution_spec__cors-invalid-origins.md_merged.snap

```diff
@@ -0,0 +1,11 @@
+---
+source: tests/execution_spec.rs
+expression: merged
+---
+schema @server(headers: {corsParams: {allowCredentials: true, allowHeaders: null, allowMethods: null, allowOrigins: ["*"], allowPrivateNetwork: false, exposeHeaders: [], maxAge: null, vary: ["origin", "access-control-request-method", "access-control-request-headers"]}}) @upstream(baseURL: "http://example.com", batch: {delay: 1, headers: [], maxSize: 1000}) {
```

Comment:
lot of empty fields with nulls and empty array. Use `skip_serialization` attribute to drop unnecessary default values from it's encoding.

Reply:
done